cegs_portal/search/templates/search/v1/partials/_multi_experiment_index.html

```diff
@@ -34,23 +34,147 @@
+                      type="button"
+                      data-twe-collapse-init
+                      data-twe-collapse-collapsed
+                      data-twe-target="#collapseThree"
+                      aria-expanded="false"
+                      aria-controls="collapseThree">
+                      <div class="text-xl font-bold text-slate-500">
+                        <i class="bi bi-download mr-1"></i> Download Data from Selected Experiments
+                      </div>
+                      <span
+                        class="-me-1 ms-auto h-5 w-5 shrink-0 rotate-[-180deg] transition-transform duration-200 ease-in-out group-data-[twe-collapse-collapsed]:me-0 group-data-[twe-collapse-collapsed]:rotate-0 motion-reduce:transition-none [&>svg]:h-6 [&>svg]:w-6">
+                        <svg
+                          xmlns="http://www.w3.org/2000/svg"
+                          fill="none"
+                          viewBox="0 0 24 24"
+                          stroke-width="1.5"
+                          stroke="currentColor">
+                          <path
+                            stroke-linecap="round"
+                            stroke-linejoin="round"
+                            d="M19.5 8.25l-7.5 7.5-7.5-7.5" />
+                        </svg>
+                      </span>
+                    </button>
+                  </h2>
+                  <div
+                    id="collapseThree"
+                    class="!visible hidden"
+                    data-twe-collapse-item
+                    aria-labelledby="headingThree"
+                    data-twe-parent="#accordionExample">
+                    <div class="px-5 py-4">
+                        <form name="dataDownloadForm">
+                            <div>
+                                <label class="font-bold flex justify-center text-center" for="dataDlFile">
+                                    Select a .bed File with Regions of Interest
+                                </label>
+                                <input class="hidden" id="dataDownloadInput" type="file" accept=".bed" name="dataDlFile" />
+                                <label for="dataDownloadInput" class="flex justify-center text-center global-button mt-3 border-2 p-2 text-sm">
+                                    Upload file
+                                </label>
+                            </div>
+                        </form>
+                        <div class="italic mt-5 mb-5 flex justify-center text-center">
+                            Please select at least one experiment.
+                        </div>
+                    </div>
```

Comment:
This partial shouldn't have the download data UI

Reply:
I think the user might like to be able to use this feature since they are already selecting mult exp and analyzing them.  It may confuse them if they have to figure out the only way to download is if they go to the exp list page. I will remove it and maybe it can be up for conversation later.